crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -390,6 +400,11 @@ pub fn check_visibility(
                     return;
                 }
 
+                let use_far_culling = match maybe_perspective_projection {
```

Comment:
This can be reduced via `Option::unwrap_or`.

Reply:
In this case is that actually a reduction? 

I can't see a way to get unwrap or to work without needlessly creating a projection.

Match seems more explicit in this case

Reply:
You can use a .map to get the use_far_culling from the Some case and unwrap_or(false) on the resulting enum

Reply:
With the suggestion to make the `far` member into an `Option<f32>` this would be unnecessary and could just be `far.is_some()`

Reply:
when making far optional, how would we want to handle the following?

example from: `crates\bevy_render\src\camera\projection.rs`
``` rs
impl CameraProjection for OrthographicProjection {
    fn get_projection_matrix(&self) -> Mat4 {
        Mat4::orthographic_rh(
            self.area.min.x,
            self.area.max.x,
            self.area.min.y,
            self.area.max.y,
            // NOTE: near and far are swapped to invert the depth range from [0,1] to [1,0]
            // This is for interoperability with pipelines using infinite reverse perspective projections.
            self.far, // .unwrap_or(1000.0) ?
            self.near,
        )
    }
```

defaulting to 1000.0 seems sensible for keeping previous behavior but also confusing?

Reply:
I would expect using 1000.0 yes, maybe extract it to a constant named DEFAULT_FAR to make it clearer.

Reply:
This no longer seems relevant?

Reply:
Right now it uses unwrap_or_default() so it will result in 0. But all default far values are set like they were previously so it most likely won't break anything.